source/ranges.tex

```diff
@@ -6372,12 +6383,16 @@
 namespace std::ranges {
   template<@\libconcept{input_range}@ V, size_t N>
     requires @\libconcept{view}@<V> && @\exposconcept{has-tuple-element}@<range_value_t<V>, N> &&
-             @\exposconcept{has-tuple-element}@<remove_reference_t<range_reference_t<V>>, N>
+             @\exposconcept{has-tuple-element}@<remove_reference_t<range_reference_t<V>>, N> &&
+             @\exposconcept{returnable-element}@<range_reference_t<V>, N>
   template<bool Const>
   class elements_view<V, N>::@\exposid{iterator}@ {                 // \expos
     using @\exposid{Base}@ = @\exposid{maybe-const}@<Const, V>;                 // \expos
 
-    iterator_t<@\exposid{Base}@> @\exposid{current_}@ = iterator_t<@\exposid{Base}@>();
+    iterator_t<@\exposid{Base}@> @\exposid{current_}@ = iterator_t<@\exposid{Base}@>();     // \expos
```

Comment:
4e228576e2ef3a4280404904c0b51c4f788c7718 should've been a "fixup:" commit, I think.